.gitignore

```diff
@@ -29,6 +29,8 @@ build/
 TestResults/
 test_failures.txt
 zone*export.txt
+testcer.cer
```

Comment:
Is it necessary to modify these .gitignore/testkey.pvk or azure-cli1.sln? I remember the live test is run automatically so we don't need to modify these tests. (Correct me if I'm wrong)